build-utils/src/index.ts

```diff
@@ -17,11 +17,13 @@ export interface Logger {
 export interface BuildOptions {
   skipNpmInstall: boolean;
   logger: Logger;
+  outputToErr: boolean;
```

Comment:
I think this will be a bit more flexible
```suggestion
  spawnOptions?: SpawnOptions,
```
we already know of a third option we for stdio (in the builder-function context)

You can import `SpawnOptions` from `child_process`

You can use "object spread" for this:
``` js
x = { a: 1, b: 2 }
y = {
  a: 3,
  c: 5,
  ...x,
}
// y => { a: 1, c: 5, b: 2 }
```